app/src/test/java/com/example/benchproject/app/movie/details/viewModel/MovieDetailsViewModelTest.kt

```diff
@@ -97,8 +98,8 @@ internal class MovieDetailsViewModelTest {
 
     @Test
     fun `when getMovieDetailsUseCase called and returns throwable isLoaderVisible should be true`() = runTest {
-        coEvery { getMovieDetailsUseCase(movieId) } returns badResponse
-        every { movieDetailsFragmentNavigator.errorSnackBar(R.string.errorMessageMovies) } just Runs
+        coEvery { getMovieDetailsUseCase(movieId) } returns badResponse andThen Result.success(movieDetailsModel)
+        every { movieDetailsFragmentNavigator.errorSnackBar(R.string.errorMessageMovies, onAction = capture(slot)) } answers { slot.captured.invoke() }
```

Comment:
Why you are invoking "onAction" here? Seems that test verifies only `isLoaderVisible` and no calls on snack bar are needed? copy-paste fail? :)
Check other places as well

Reply:
Yes, it is not necessary and I changed it :D